subprojects/shipkit/src/test/groovy/org/shipkit/internal/gradle/java/tasks/CreateDependencyInfoFileTest.groovy

```diff
@@ -0,0 +1,95 @@
+package org.shipkit.internal.gradle.java.tasks
+
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.artifacts.ModuleVersionIdentifier
+import org.gradle.api.artifacts.ResolvedArtifact
+import org.gradle.api.artifacts.ResolvedConfiguration
+import org.gradle.api.artifacts.ResolvedModuleVersion
+import testutil.PluginSpecification
+
+class CreateDependencyInfoFileTest extends PluginSpecification {
+
+    CreateDependencyInfoFile createDependencyInfoFile = new CreateDependencyInfoFile()
+
+    def "should create correct output file"() {
+        given:
+        def configuration = Mock(Configuration)
```

Comment:
Alternatively, you can add jcenter and resolve the dependencies normally. It would make the test easier to read and probably to maintain, too.

```
project.repositories.jCenter()
def conf = project.configurations.create("conf")
project.dependencies {
  conf "junit:junit:4.10"
  conf "org.mockito:mockito-core:2.10.0"
}
```

Using real dependencies will also allow you to assert on the transitive dependencies (which I leave up to you to decide if we want to support them - I'm ok with that).